api/openapi.yaml

```diff
@@ -1002,6 +1002,18 @@ components:
         speed_limit_composition:
           type: string
           description: Train category for speed limits
+        margin_before:
```

Comment:
From a user point of view, margin should be added like standard margins, which can be percent, time, timePerDistance. See #/components/schemas/AllowanceValue and check with PO. Thanks !

Reply:
This is not like the other margins, it can't be a percent or relative to distance. This only makes sense as a flat value.